notebooks/peerscout/peerscout-get-editor-pubmed-paper-ids.ipynb

```diff
@@ -76,103 +79,226 @@
+    "\n",
+    "print(\"existing_editor_ids_and_pubmed_url_df length: \", len(existing_editor_ids_and_pubmed_url_df))\n",
+    "existing_editor_ids_and_pubmed_url_df.head(3)"
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
-    "existing_editor_ids_set = set(existing_editor_ids_df['person_id'])\n",
-    "len(existing_editor_ids_set)"
-   ]
+    "existing_editor_ids_set = set(existing_editor_ids_and_pubmed_url_df['person_id'])\n",
+    "print(\"existing_editor_ids_set length :\", len(existing_editor_ids_set))"
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "editor_parsed_pubmed_links_df = read_big_query(\n",
     "    'SELECT * FROM `{table_name}`\\nWHERE parsed_search_term IS NOT NULL'.format(\n",
     "        table_name=editor_parsed_pubmed_links_table_name\n",
     "    )\n",
     ")\n",
-    "print(len(editor_parsed_pubmed_links_df))\n",
-    "editor_parsed_pubmed_links_df.head()"
-   ]
+    "print(\"editor_parsed_pubmed_links_df length: \", len(editor_parsed_pubmed_links_df))\n",
+    "editor_parsed_pubmed_links_df.head(3)"
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "source": [
+    "merged_editor_parsed_pubmed_links_df = editor_parsed_pubmed_links_df.merge(\n",
+    "    existing_editor_ids_and_pubmed_url_df,\n",
+    "    how='left',\n",
+    "    on='person_id',\n",
+    "    suffixes=('', '_existing')\n",
+    ")\n",
+    "print(\"merged_editor_parsed_pubmed_links_df length: \", len(merged_editor_parsed_pubmed_links_df))\n",
+    "merged_editor_parsed_pubmed_links_df.head(3)"
+   ],
    "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "source": [
-    "remaining_editor_parsed_pubmed_links_df = editor_parsed_pubmed_links_df[\n",
+    "editors_with_changed_pubmed_url_df = merged_editor_parsed_pubmed_links_df[\n",
+    "    (merged_editor_parsed_pubmed_links_df['pubmed_url_existing'].notnull())\n",
+    "    &\n",
+    "    (\n",
+    "        merged_editor_parsed_pubmed_links_df['pubmed_url']\n",
+    "        !=\n",
+    "        merged_editor_parsed_pubmed_links_df['pubmed_url_existing']\n",
+    "    )\n",
+    "].drop(columns=['pubmed_url_existing', 'imported_timestamp'])\n",
+    "\n",
+    "print(\"editors_with_changed_pubmed_url_df length: \", len(editors_with_changed_pubmed_url_df))\n",
+    "editors_with_changed_pubmed_url_df.head(3)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "editors_with_not_currently_updated_info_df = merged_editor_parsed_pubmed_links_df[\n",
+    "    (\n",
+    "        pd.Timestamp.now(tz=pytz.UTC)\n",
```

Comment:
this probably should be `pd.Timestamp.utcnow()`? Otherwise I guess it will be a local time with the timezone of UTC?